test/test_maxwellDriver1d.py

```diff
@@ -1057,3 +1058,50 @@ def test_buildDrivedEvolutionOperator():
     plt.spy(A_drived, markersize=4)
     # plt.spy(A, markersize=4)
     plt.show()
+
+def test_check_initial_conditions_GW_right():
+
+    x_min = -4.0
+    x_max = 4.0
+    k_elements = 400
+
+    sp = FDTD1D(mesh=Mesh1D(x_min, x_max, k_elements, boundary_label="PEC"))
+    driver = MaxwellDriver(sp, timeIntegratorType='LF2', CFL=1.0)
+
+    s0 = 0.25
+    initialFieldE = np.exp(-(sp.x)**2/(2*s0**2))
+    initialFieldH = np.exp(-(sp.xH)**2/(2*s0**2))
+    
+    driver['E'][:] = initialFieldE[:]
+    driver['H'][:] = initialFieldH[:]
+
+    x_center = int(k_elements / 2)
+    one_time_unit_elements = int(k_elements / (x_max - x_min))
+
+    E_gaussian_center_start = driver['E'][x_center]
+    H_gaussian_center_start = driver['H'][x_center-1]    
+
+    driver.run_until(1.0)
+
+    E_gaussian_center_end = driver['E'][x_center+one_time_unit_elements]
```

Comment:
Checking against a single value maybe is not the best idea. Try to find an expression for how the whole field should look like at that time step and compare with that. You can use the squared error or something like that to compare.